src/pkg/utils/utils.go

```diff
@@ -44,6 +44,7 @@ type Distro struct {
 	ContainerNamePrefix    string
 	ImageBasename          string
 	ParseRelease           ParseReleaseFunc
+	ReleaseFormat          string
```

Comment:
I think it will be better to use the errors thrown by a `ParseReleaseFunc` to describe what was wrong with the `release` option, than to have a one-size-fits-all-errors string.  The `ParseReleaseFunc` knows exactly what went wrong while parsing, so it can clearly present the problem to the user.

A custom `ParseReleaseError` with a `hint` string to describe the problem in detail might work very well here.  See my comment about sentinels and custom `error` implementations.

The `toolbox(1)` manual already has a list of accepted `release` formats for each `distro` with examples.  It seems like the `ReleaseFormat` string is a more terse representation of that.  This means that accepted `release` formats are described in three different places:
* in the manual
* in the `ReleaseFormat` string
* in the `ParseReleaseFunc`

It will be good if we could reduce that to two.  :)

Reply:
https://github.com/containers/toolbox/pull/1100 uses the errors thrown by the `ParseReleaseFunc` implementations.

Reply:
https://github.com/containers/toolbox/pull/1101 introduces a `ParseReleaseError`.